packages/handlers/odata/src/declarations.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'string-interpolation/src';
```

Comment:
Can you try to use `dotjs` instead of `string-interpolation` in this PR? 
We want to change it anyway, so it might be a good chance to try it.

Reply:
It has different syntax/delimiters and stuff. This might cause a inconsistency between handlers. It would be better to do this replacement in a seperate better in my opinion.